### ISSUE TITLE
Sound Input

### ISSUE THREAD

**Issue by joyhughes** [joyhughes/Jen#167]
Create functions that respond to sound, either microphone input or audio clip. 
Functions will give the amplitude over a given frequency band.
Image output can then respond to the sound.

**Comment by DM-netizen** [comment]
Hello @joyhughes 
Can I take up this issue?

**Comment by joyhughes** [comment]
Yes, but I'm not sure exactly how to do it. We must find a cross platform open source library to get the sound input. A Jen function can be constructed that returns a floating point value that approximates the amplitude over a given frequency range and interval of time.
________________________________

**Comment by joyhughes** [comment]
We can probably get audio from the web browser https://web.dev/articles/media-recording-audio

Then process in C++.

Doing it in real time is a challenge
________________________________

**Comment by DM-netizen** [comment]
Okk
I will try to implement it

**Comment by joyhughes** [comment]
I think a good way to start purely in C++ is to load an audio file from built-in storage (the same way it loads images, from lux_files directory).
________________________________
